src/Nox.Reference.Data.World/README.md

```diff
@@ -1,9 +1,84 @@
-In order to create migrations:
+#Nox.Refence.World project
+
+#How to use:
+There are 2 approaches how to deal with Nox.Refence.World:
+	1. Static approach: 
+		Allow to call extension methods or run query over quryable properties.
+		Currency currency = World.Currencies.Get("USD");
```

Comment:
we can improve formatting by adding code pieces here instead of just text. Plus add a little bit more info and mention method documentation

Reply:
we can improve formatting by adding code pieces here instead of just text. 
What do you mean here. 
There is simple usage of method.
Currency currency = World.Currencies.Get("USD");